mesheryctl/pkg/utils/fixtures/platform/manifesturl.api.response.golden

```diff
@@ -5010,170 +5010,6 @@
-},
-{
-"path": "docs/assets/webfonts/fa-regular-400.woff2",
-"mode": "100644",
-"type": "blob",
-"sha": "b9344a742ff2089c6bf862dcb92700cbed31cf0b",
-"size": 13580,
-"url": "https://api.github.com/repos/meshery/meshery/git/blobs/b9344a742ff2089c6bf862dcb92700cbed31cf0b"
-},
-{
-"path": "docs/assets/webfonts/fa-solid-900.eot",
-"mode": "100644",
-"type": "blob",
-"sha": "c77baa8d46ab45aff98845588c807e81b73c9d91",
-"size": 192116,
-"url": "https://api.github.com/repos/meshery/meshery/git/blobs/c77baa8d46ab45aff98845588c807e81b73c9d91"
-},
-{
-"path": "docs/assets/webfonts/fa-solid-900.svg",
-"mode": "100644",
-"type": "blob",
-"sha": "96520bcd8d35624179187736f82670ae1f6508a1",
-"size": 827499,
-"url": "https://api.github.com/repos/meshery/meshery/git/blobs/96520bcd8d35624179187736f82670ae1f6508a1"
-},
-{
-"path": "docs/assets/webfonts/fa-solid-900.ttf",
-"mode": "100644",
-"type": "blob",
-"sha": "c6c3dd4d40e6935dbda5f4fc671956c41b768a09",
-"size": 191832,
-"url": "https://api.github.com/repos/meshery/meshery/git/blobs/c6c3dd4d40e6935dbda5f4fc671956c41b768a09"
-},
-{
-"path": "docs/assets/webfonts/fa-solid-900.woff",
-"mode": "100644",
-"type": "blob",
-"sha": "77c1786227f5377318cf4f52fff9f454874651b5",
-"size": 98020,
-"url": "https://api.github.com/repos/meshery/meshery/git/blobs/77c1786227f5377318cf4f52fff9f454874651b5"
-},
-{
-"path": "docs/assets/webfonts/fa-solid-900.woff2",
-"mode": "100644",
-"type": "blob",
-"sha": "e30fb671128fdf2f4fc6d7b1c96c0736bc547e58",
-"size": 75440,
-"url": "https://api.github.com/repos/meshery/meshery/git/blobs/e30fb671128fdf2f4fc6d7b1c96c0736bc547e58"
-},
-{
-"path": "docs/lib",
-"mode": "040000",
-"type": "tree",
-"sha": "2cde1d2ca9959533f963932a55b800a55a3d6a9b",
-"url": "https://api.github.com/repos/meshery/meshery/git/trees/2cde1d2ca9959533f963932a55b800a55a3d6a9b"
-},
-{
-"path": "docs/lib/tasks",
-"mode": "040000",
-"type": "tree",
-"sha": "aeaee88dc82a29173e20c8223276dcd82dc4bb07",
-"url": "https://api.github.com/repos/meshery/meshery/git/trees/aeaee88dc82a29173e20c8223276dcd82dc4bb07"
-},
-{
-"path": "docs/lib/tasks/helper.rb",
-"mode": "100644",
-"type": "blob",
-"sha": "0e70724b427c63809a944fedacb80968a75a199d",
-"size": 15168,
-"url": "https://api.github.com/repos/meshery/meshery/git/blobs/0e70724b427c63809a944fedacb80968a75a199d"
-},
-{
-"path": "docs/lib/tasks/renderer.rb",
-"mode": "100644",
-"type": "blob",
-"sha": "87308646ac0e632b90fdbb6ef386edabe8254365",
-"size": 1425,
-"url": "https://api.github.com/repos/meshery/meshery/git/blobs/87308646ac0e632b90fdbb6ef386edabe8254365"
-},
-{
-"path": "docs/lib/tasks/templates",
-"mode": "040000",
-"type": "tree",
-"sha": "cfeec97ced6294242a1344b90d1cd5d7c500edf8",
-"url": "https://api.github.com/repos/meshery/meshery/git/trees/cfeec97ced6294242a1344b90d1cd5d7c500edf8"
-},
-{
 "path": "docs/lib/tasks/templates/default.md.haml",
 "mode": "100644",
 "type": "blob",
```

Comment:
But still I'm totally surprised that these golden files have entire mock of GitHub API which includes the whole file list of meshery repo. This one must be looked on ＞︿＜